cli/compose/loader/volume.go

```diff
@@ -23,12 +26,13 @@ func parseVolume(spec string) (types.ServiceVolumeConfig, error) {
 	}
 
 	buffer := []rune{}
-	for _, char := range spec {
+	for _, char := range spec + string(endOfSpec) {
```

Comment:
That's kind of what the old code did. Instead of using this token it would just repeat this case outside of the for loop. We still need a token to pass to `populateFieldFromBuffer()` to identify "end of string", so I realized it was actually easier to just append the token to the string and add it to the case.